web/src/app/schedules/calendar-subscribe/CalendarSubscribeEditDialog.tsx

```diff
@@ -36,27 +43,31 @@ export function CalendarSubscribeEditDialogContent(props) {
   })
 
   // setup the mutation
-  const [updateSubscription, updateSubscriptionStatus] = useMutation(mutation, {
-    variables: {
-      input: {
-        id: props.data.id,
-        name: value.name,
-        fullSchedule: value.fullSchedule,
-      },
-    },
-    onCompleted: () => props.onClose(),
-  })
+  const [updateSubscriptionStatus, commit] = useMutation(mutation)
 
   return (
     <FormDialog
       title='Edit Calendar Subscription'
       onClose={onClose}
-      loading={updateSubscriptionStatus.loading}
-      onSubmit={() => updateSubscription()}
+      loading={updateSubscriptionStatus.fetching}
+      onSubmit={() =>
+        commit(
+          {
+            input: {
+              id: props.data.id,
+              name: value.name,
+              fullSchedule: value.fullSchedule,
+            },
+          },
+          { additionalTypenames: ['User'] },
```

Comment:
```suggestion
          { additionalTypenames: ['User', 'UserCalendarSubscription'] },
```
I believe this should include `UserCalendarSubscription` to avoid the update lag. You can replicate this if you edit a subscription's title and then immediately edit it again - it'll show the old title for 2-5 seconds in the edit dialog before it updates to the new value.

Reply:
I'm not seeing this behavior- if I edit and save it updates after just a moment for me

Reply:
@Forfold that's likely just timing, since our update mutations return a bool -- there is no immediate update _unless_ we add the object type here.

most likely without it we'll see flaky tests. 2-5 seconds usually means the catch-all-refetch is the only reason you're seeing the correct data come in

it should update almost instantly locally if it's configured correctly.

Reply:
This information is nested under the User query though, no? This query is only used by the edit dialog by the looks of it, which closes and resets all fields on complete

Reply:
After testing, both solutions work just fine, but we don't need both. I pushed up the requested change. @mastercactapus @tony-tvu